nrf_security/src/mbedcrypto_glue/cc310/cc310_glue.cmake

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+kconfig_mbedtls_debug("######### Creating cc310 glue library #########")
+
+if (CONFIG_GLUE_MBEDTLS_AES_C AND CC310_MBEDTLS_AES_C)
+  set(GLUE_CC310_MBEDTLS_AES_C TRUE)
+  kconfig_mbedtls_debug("cc310 backend glue: AES")
+endif()
+
+if (CONFIG_GLUE_MBEDTLS_CCM_C AND CC310_MBEDTLS_CCM_C)
+  set(GLUE_CC310_MBEDTLS_CCM_C TRUE)
+  kconfig_mbedtls_debug("cc310 backend glue: CCM")
+endif()
+
+if (CONFIG_GLUE_MBEDTLS_DHM_C AND CC310_MBEDTLS_DHM_C)
+  set(GLUE_CC310_MBEDTLS_DHM_C TRUE)
+  kconfig_mbedtls_debug("cc310 backend glue: DHM")
+endif()
+
+
+zephyr_library_named(mbedcrypto_glue_cc310)
+
+#
+# Adding cc310 backend glue files
+#
+zephyr_library_sources_ifdef(GLUE_CC310_MBEDTLS_AES_C
+  ${CMAKE_CURRENT_LIST_DIR}/aes_cc310.c
```

Comment:
drop the CMAKE_CURRENT_LIST_DIR

Reply:
Can't get it to work without CMAKE_CURRENT_LIST_DIR. Remember that this was added with include and not add_subdirectory...

Reply:
ah, right